vm/systemSmartContracts/delegation.go

```diff
@@ -988,7 +1017,9 @@ func (d *delegation) reDelegateRewards(args *vmcommon.ContractCallInput) vmcommo
 
 	delegateValue := big.NewInt(0).Set(delegator.UnClaimedRewards)
 	delegator.UnClaimedRewards.SetUint64(0)
-	return d.finishDelegateUser(globalFund, delegator, dConfig, dStatus, args.CallerAddr, args.RecipientAddr, delegateValue, false)
+
+	return d.finishDelegateUser(globalFund, delegator, dConfig, dStatus, args.CallerAddr,
```

Comment:
I would rename the `CheckCapOnReDelegateRewards` because when I first read it I thought is a function pointer or something. Maybe `FlagCheckCapOnReDelegateRewards` ?

Reply:
this is not a flag. it is a propriety.

Reply:
Sounds like a function